Tests/OptimizelyTests-DataModel/SemanticVersionTests.swift

```diff
@@ -118,4 +118,41 @@ class SemanticVersionTests: XCTestCase {
         
         XCTAssert(version.compareVersion(targetedVersion: target) < 0)
     }
+    
+    func testBetaTagGreater() {
+        let target = "2.1.2"
+        let version = "2.1.3-beta"
+        
+        XCTAssert(version.compareVersion(targetedVersion: target) > 0)
+    }
+    
+    func testBetaToRelease() {
+        let target = "2.1.2-release"
+        let version = "2.1.2-beta"
+        
+        XCTAssert(version.compareVersion(targetedVersion: target) < 0)
```

Comment:
Is this test valid? I thought "-release" cannot be in the target?

Reply:
both of those are valid suffixes in the semver pre-release www.semver.org